diff --git a/dist/source/index.js b/dist/source/index.js
index cae5fcbd08b52f9de9e6198f3af9cf0ccce850ac..51368cb067ddbfb219e3544790ef7ec13ebfc9f0 100644
--- a/dist/source/index.js
+++ b/dist/source/index.js
@@ -539,11 +539,7 @@ export default class Conf {
             ...options.ajvOptions,
         });
         ajvFormats(ajv);
-        const schema = {
-            ...options.rootSchema,
-            type: 'object',
-            properties: options.schema,
-        };
+        const schema = options.schema;
         this.#validator = ajv.compile(schema);
         this.#captureSchemaDefaults(options.schema);
     }
